### ISSUE TITLE
Two buttons instead of the drop-down

### ISSUE THREAD

**Issue by hitzu111** [krzychu124/Traffic#9]
There's plenty of space on the top bar for one more button. An extra click to get to the right tool is just annoying TBH. Consider 2 separate buttons for each of the tools: Lane Connector Tool and Priorities Tool

**Comment by krzychu124** [comment]
> There's plenty of space on the top bar for one more button. An extra click to get to the right tool is just annoying TBH. Consider 2 separate buttons for each of the tools: Lane Connector Tool and Priorities Tool

Are you aware you can use key binding to enable/switch between tools (each has own binding for toggle)? You don't need to use UI almost for anything, 90% of actions can be done with keybindings

**Comment by hitzu111** [comment]
Yes of course I am :) It's not like I want to play piano while playing the game though :D

**Comment by krzychu124** [comment]
I'll check what can be done

**Comment by irasponsible** [comment]
I came here to suggest the same thing - having it as a separate button would be much easier.
